### DISCUSSION TITLE
Dịch equivariance

### DISCUSSION THREAD

**Discussion by minhduc0711** [mlbvn/handson-ml2-vn#143] [Terminology] [answered]
Context:
> If the input image is translated by one pixel to the right, the output should also be translated by one pixel to the right. The goal in this case is equivariance, not invariance: a small change to the inputs should lead to a corresponding small change in the output.

"tương đương biến" nghe có vẻ hơi buồn cười :v

@mlbvn/handson-ml

**Comment by lkhphuc** [comment]
Mình hay hiểu là:
- Variance: Phương sai ( ... biến ?) 
- Covariance: Đồng biến / Hiệp biến
- Invariance: Bất biến
- Equivariance: Đẳng biến

**Comment by ngcthuong** [comment]
Đẳng biến có vẻ không phù hợp vì có từ "đẳng" tức là bằng ở đây. Ở đây ám chỉ nó cũng thay đổi một lượng "tương ứng" với đầu vào, chứ không phải một lượng giống nhau. 

Nghữ nghĩa thì có vẻ hợp với **đồng biến** hơn, tuy nhiên lại lặp với Covariance.

**Reply by minhduc0711** [reply]
Mình chốt như suggestion của anh Thướng nhé.

**Reply by lkhphuc** [reply]
Mình hiểu tiền tố `equi` trong `equivalence` và `equivariance` là chung một gốc, nên như a Thướng nói dịch là `đẳng` đúng hơn chứ nhỉ.
Formal thì equivariance là đẳng thức `f(T(x)) = T(f(x))`.

**Reply by thanhcsf** [reply]
cc @minhduc0711 @ngcthuong anh em trao đổi thêm một chút nữa đã nhé.

**Reply by ngcthuong** [reply]
Mình có tìm thêm, thấy từ equuvarient đã đc dịch khá khổ biến là đẳng biến. Ngoài ra định nghĩa gốc hay dùng trong neural net có vẻ cũng dùng được nghĩa đẳng biến. Mình vote lại là đẳng biến nhé.

**Comment by thanhcsf** [comment, marked as answer]
Vậy chỗ này mình chốt lại **equivariance** sẽ dịch là **đẳng biến** nhé.
